Makefile

```diff
@@ -73,6 +73,9 @@ test-zeek: bin/$(ZEEKPATH)
 perf-compare: build $(SAMPLEDATA)
 	scripts/comparison-test.sh
 
+zng-output-check: build $(SAMPLEDATA)
+	scripts/zng-output-check.sh
```

Comment:
Not sure how others feel, but I'd prefer this over a separate script.
```mk
zng-output-check: build $(SAMPLEDATA)
	PATH=$(CURDIR)/dist:$PATH zq-sample-data/scripts/check_md5sums.sh tzng
	PATH=$(CURDIR)/dist:$PATH zq-sample-data/scripts/check_md5sums.sh zng
	PATH=$(CURDIR)/dist:$PATH zq-sample-data/scripts/check_md5sums.sh zng-uncompressed
```

Reply:
I like the script since it gives more documentation space that would clutter the makefile.